### DISCUSSION TITLE
Consume Chatflow in Slack

### DISCUSSION THREAD

**Discussion by michael-elkabetz** [FlowiseAI/Flowise#1808] [General]
Is it possible to make conversion with slack bot (PM) that send questions to FlowiseAI Q&A Chatflow and sending answers back to sender??

**Comment by Jaredude** [comment]
The short answer is: yes. The long answer is that it's not a simple process. It's certainly possible to accomplish this, but it's not going to be a straightforward process of just dragging and dropping a Slack node into Flowise. Making a Slack App comes with very specific setup requirements, and Slack also has a 3000ms response requirement that you'd need to code around because many AI responses will take much longer than 3 seconds to respond. You can read up on the [Slack API here](https://api.slack.com/start/apps).
